media/css/custom.scss

```diff
@@ -0,0 +1,496 @@
+      }
+    }
+  }
+  .caption {
+    p {
+      font-size: 14px;
+    }
+  }
+}
+
+.twitter-follow-button {
+  margin-top: 3px;
+}
+
+.btn-danger {
+  background: $sidebar-link-highlight;
+  &:hover {
+    background: $danger-link-highlight;
+  }
+}
+
+h1,
+h2,
+h3 {
+  color: $dark-blue;
+}
+
+.leftimg,
+.deanimg {
+  height: 300px;
+}
+
+.leftimg {
+  float: left;
+  margin: 10px 10px 20px 0;
+}
+
+.deanimg {
+  margin: 20px 100px 20px 0;
+}
+
+.codehilite {
+  clear: left;
+}
+
+.badge {
+  padding-left: 6px;
+  padding-bottom: 3px;
+  font-size: 11px;
+}
+
+.badge-important {
+  background-color: $sidebar-link-highlight;
+}
+
+#loader {
+  background-color: $sidebar-link-highlight;
+  margin-bottom: 20px;
+  padding-bottom: 3px;
+}
+
+#spinner {
+  display: inline;
+  margin-right: 3px;
+}
+
+*::selection,
+*::-moz-selection,
+*::-webkit-selection {
+  background: $bright-red;
+  color: $white;
+}
+
+@import "social_buttons";
+@import "navigation";
+
+* {
+  // old-style reset here
+  border: 0;
+  padding: 0;
+}
+
+table {
+  border-collapse: separate;
+  border: 1px solid $grayish-blue;
+  border-width: 0 0 1px 1px;
+  font-size: 20px;
+}
+
+th {
+  text-align: center;
+  vertical-align: middle;
+  position: relative;
+}
+
+th {
+  font-weight: bold;
+  font-size: 14px;
+}
+
+td:hover,
+th:hover {
+  background-position: 0 -81px;
+  color: $light-black;
+}
+
+.events {
+  position: relative;
+  ul {
+    text-align: left;
+    position: absolute;
+    display: none;
+    z-index: 1000;
+    padding: 15px;
+    background: $light-grayish-blue;
+    color: $white;
+    border: 1px solid $white;
+    font-size: 15px;
+    width: 600px; // 200px, wow banner
+    @include border-radius(3px);
+    list-style: none;
+    color: $gray;
+    -webkit-box-shadow: 0 8px 8px $dark-gray;
+    box-shadow: 0 8px 8px $dark-gray;
+  }
+  li {
+    padding-bottom: 5px;
+    span {
+      display: block;
+      font-size: 12px;
+      text-align: justify;
+      color: $gray;
+      &.title {
+        font-weight: bold;
+        color: $light-black;
+      }
+    }
+  }
+}
+
+// applicants help
+
+.table-help-inf {
+  font-size: 13px;
+  width: 572px;
+  border-collapse: collapse;
+  margin-top: 30px;
+  border: 1px solid;
+  th {
+    border: 1px solid;
+  }
+  td {
+    border: 1px solid $light-gray;
+    padding-left: 3px;
+  }
+}
+
+.table-help {
+  text-align: center;
+}
+
+// student council
+
+.i2Style {
+  font: bold 24px Tahoma, Geneva, sans-serif;
+  color: $white;
+  border: 0 solid $white;
+  text-shadow: 0 -1px 1px $light-black;
+  box-shadow: 1px -12px 0 $moderate-blue;
+  -moz-box-shadow: 1px -12px 0 $moderate-blue;
+  -webkit-box-shadow: 1px -12px 0 $moderate-blue;
+  width: 575px;
+  padding: 10px;
+  cursor: pointer;
+  margin: 0 auto;
+  &:active {
+    cursor: pointer;
+    position: relative;
+    top: 2px;
+  }
+  ~ img {
+    margin-bottom: 35px;
+  }
+}
+
+.widget-item {
+  height: 180px;
+  padding-right: 0;
```

Comment:
Properties should be ordered height, opacity, padding-bottom, padding-left, padding-right, position, -moz-transition, -ms-transition, -o-transition, -webkit-transition, transition